packages/common/src/services/remote-config/defaults.ts

```diff
@@ -31,7 +31,8 @@ export const remoteConfigIntDefaults: { [key in IntKeys]: number | null } = {
   [IntKeys.BUY_AUDIO_WALLET_POLL_MAX_RETRIES]: 120,
   [IntKeys.BUY_AUDIO_SLIPPAGE]: 3,
   [IntKeys.GATED_TRACK_POLL_INTERVAL_MS]: 1000,
-  [IntKeys.DISCOVERY_NOTIFICATIONS_GENESIS_UNIX_TIMESTAMP]: 0
+  [IntKeys.DISCOVERY_NOTIFICATIONS_GENESIS_UNIX_TIMESTAMP]: 0,
+  [IntKeys.CACHE_ENTRY_TTL]: 0
```

Comment:
is 0 a safe default? Would that invalidate everything immediately?

Reply:
what's potentially really silly, is that this makes it falsy, so the DEFAULT value in the code gets used. BUT now that you have me thinking, we should just get rid of the constant and use this.